src/ctl/plugins/pypi.py

```diff
@@ -105,7 +105,17 @@ def _build_dist(self, **kwargs):
         Build dist
         """
 
-        command = ["rm dist/* -rf", "python setup.py sdist"]
+        command = ["rm dist/* -rf"]
+
+        path = self.repository.checkout_path
+
+        if os.path.exists(os.path.join(path, "setup.py")):
+            command.append("python setup.py sdist")
+        elif os.path.exists(os.path.join(path, "poetry.lock")):
+            command.append("poetry build -f sdist")
```

Comment:
Is there a reason we're limiting to sdist?

Reply:
hrm, i dont recall, it was what i had before, but i guess we can build wheel as well